### PR TITLE
Generate jdbc sql

### PR DESCRIPTION
**Pull request by ChengJie1053**
Generate jdbc sql

### CODE REVIEW

File: linkis-public-enhancements/linkis-datasource/linkis-metadata-query/server/src/main/java/org/apache/linkis/metadata/query/server/restful/MetadataQueryRestful.java [hunk 1]
```diff
@@ -467,6 +467,62 @@ public Message getSparkSql(
...
+
+      String userName =
+          ModuleUserUtils.getOperationUser(request, "getJdbcSql, dataSourceName:" + dataSourceName);
+
+      GenerateSqlInfo sparkSql =
+          metadataQueryService.getJdbcSqlByDsNameAndEnvId(
+              dataSourceName, database, table, system, userName, envId);
+      return Message.ok().data("jdbcSql", sparkSql);
+    } catch (Exception e) {
+      return errorToResponseMessage(
```
Comment (peacewong) [inline linkis-public-enhancements/linkis-datasource/linkis-metadata-query/server/src/main/java/org/apache/linkis/metadata/query/server/restful/MetadataQueryRestful.java:511]: need to print log

File: linkis-public-enhancements/linkis-datasource/linkis-metadata-query/service/jdbc/src/main/java/org/apache/linkis/metadata/query/service/PostgresqlMetaService.java [hunk 2]
```diff
@@ -108,4 +109,13 @@ public List<MetaColumnInfo> queryColumns(
   public String querySqlConnectUrl(SqlConnection connection) {
     return connection.getSqlConnectUrl();
   }
+
+  @Override
+  public GenerateSqlInfo queryJdbcSql(SqlConnection connection, String database, String table) {
+    try {
+      return connection.queryJdbcSql(database, table);
+    } catch (Exception e) {
+      throw new RuntimeException("Fail to get jdbc sql (获取jdbcSql失败)", e);
```
Comment (peacewong) [inline linkis-public-enhancements/linkis-datasource/linkis-metadata-query/service/jdbc/src/main/java/org/apache/linkis/metadata/query/service/PostgresqlMetaService.java:118]: Should to use LinkisRuntimeException

Comment (ChengJie1053) [reply linkis-public-enhancements/linkis-datasource/linkis-metadata-query/service/jdbc/src/main/java/org/apache/linkis/metadata/query/service/PostgresqlMetaService.java:118]: Ok, thank you for reviewing the code

**Review by peacewong — APPROVED** [review]
LGTM.

**Pull request merged**